airbyte-integrations/connectors/source-postgres/src/test/resources/expected_spec.json

```diff
@@ -1,383 +0,0 @@
-{
```

Comment:
Deleted this file because it was unused, and I wasted some time figuring out which `expected_spec.json` file I was supposed to update